Starcounter.Uniform/FormItem/FormItemMessagesBuilder.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Starcounter.Templates;
+using Starcounter.Uniform.ViewModels;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Starcounter.Uniform.FormItem
+{
+    public class FormItemMessagesBuilder
+    {
+        private List<string> _properties = new List<string>();
+
+        /// <summary>
+        /// Specify the property to generate message structure for.
+        /// </summary>
+        /// <param name="property">Property name</param>
+        /// <returns>The original builder object</returns>
+        public FormItemMessagesBuilder ForProperty(string property)
+        {
+            _properties.Add(property);
+
+            return this;
+        }
+
+        /// <summary>
+        /// Specify the properties to generate message structure for.
+        /// </summary>
+        /// <param name="properties">The list of properties names</param>
+        /// <returns>The original builder object</returns>
+        public FormItemMessagesBuilder ForProperties(List<string> properties)
```

Comment:
always accept the broadest interface you can. In this case `IEnumerable<string>`